### PR TITLE
DAOS-7174 doc: remove duplicated sections (#7696)

### PR DESCRIPTION
**Pull request by johannlombardi**
- merge admin/installation section into the QSG
- add install scripts
- merge DAOS from scratch instructions from admin guide into QSG
- move the newly created DAOS in Docker instructions to the QSG
- move DFuse pool and system modes to specific advanced usage
  section to avoid confusion
- point at install scripts to install prerequesites when building
  from scratch

Doc-only: true

Signed-off-by: Johann Lombardi <johann.lombardi@intel.com>

### CODE REVIEW

**Comment by johannlombardi** [comment]
Not sure why, but some files had a different name on release/2.0 and master. I changed the names.

File: docs/user/filesystem.md [hunk 1]
```diff
@@ -502,3 +439,70 @@ $ D_IL_REPORT=-1 LD_PRELOAD=/usr/lib64/libioil.so dd if=/dev/zero of=./bar bs=1G
     function to close stderr on exit, so for many basic commands such as cp and cat
     whilst the interception library will work it is not possible to see the summary
```
Comment (knard-intel) [inline docs/user/filesystem.md:440]: Not in the PR, however there is probably a typo error with the word whilst

File: utils/docker/vcluster/README.md [hunk 2]
```diff
@@ -1,203 +1,4 @@
-# Using DAOS with Docker
+Docker recipes and scripts to build and deploy Docker images to simulate
+a small cluster using DAOS as backend storage.
```
Comment (knard-intel) [inline utils/docker/vcluster/README.md:2]: Maybe a reference of the page containing the full documentation could be helpful.

File: docs/user/filesystem.md (hunk 1 above)
Comment (johannlombardi) [reply docs/user/filesystem.md:440]: whilst is an english word :) I confirmed with Ashley that the phase is correct.

File: utils/docker/vcluster/README.md (hunk 2 above)
Comment (johannlombardi) [reply utils/docker/vcluster/README.md:2]: See the line below?

File: docs/user/filesystem.md (hunk 1 above)
Comment (knard-intel) [reply docs/user/filesystem.md:440]: OK, I learn a new word :)

File: utils/docker/vcluster/README.md (hunk 2 above)
Comment (knard-intel) [reply utils/docker/vcluster/README.md:2]: My bad

**Review by knard-intel — APPROVED** [review]

**Pull request merged**